contracts/test/integration/utils/TimelockSimulation.sol

```diff
@@ -127,8 +127,8 @@ contract TimelockSimulation is
             postActionVerifyOracle();
 
             /// verify mints and redeems work across all PSM's
-            postActionVerifyRedeem(vm);
             postActionVerifyMint(vm, doLogging);
+            postActionVerifyRedeem(vm);
```

Comment:
Why change the order of these out of interest?

Reply:
A test was failing that I couldn't figure out why, I reversed the order while debugging, and then never put them back. either order doesn't matter because all state changes are reverted after the minting and redemptions occur in those functions.